workflow-controller/controller-service/src/main/java/io/hyscale/controller/exception/ExitCodeExceptionMapper.java

```diff
@@ -30,7 +29,8 @@ public int getExitCode(Throwable exception) {
             return 0;
         }
         if (exception instanceof HyscaleException) {
```

Comment:
This is a good update, also can you remove the constants 1's and 0's everywhere for exit codes.